.github/workflows/tox-test.yml

```diff
@@ -24,14 +24,26 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Install Python Dependencies
-        uses: HassanAbouelela/actions/setup-python@setup-python_v1.4.0
-        id: setup_python
+      - name: Setup poetry
+        id: poetry_setup
+        uses: ItsDrike/setup-poetry@v1
         with:
-          install_args: "--with workflow-tox --without lint"
-          python_version: "${{ matrix.python-version }}"
+          install-args: "--with workflow-tox --without lint"
+          python-version: "${{ matrix.python-version }}"
+
+      - name: Pytest Cache Caching
```

Comment:
I do not really think that pytest cache should be saved in CI. After a few runs with pytest's cache, I got around 1080ms, and if append `git clean -fxd` before pytest run, it gives something around 1280ms.

Is cache restore/upload will be faster than 200ms?

Reply:
Hmm, on unix systems, a cache restore might take somewhere around 200ms, but that's certainly not the case on windows, where it can take seconds to perform a restore. I suppose it indeed makes more sense to not use this cache. Thanks!

Reply:
Removed in ad769d9